api-spec/transactions-api.yaml

```diff
@@ -451,11 +451,14 @@ components:
           oneOf:
             - $ref: "#/components/schemas/PostePayAuthRequestDetails"
             - $ref: "#/components/schemas/CardAuthRequestDetails"
+            - $ref: "#/components/schemas/NpgAuthRequestDetails"
           discriminator:
             propertyName: detailType
             mapping:
               postepay: "#/components/schemas/PostePayAuthRequestDetails"
               card: "#/components/schemas/CardAuthRequestDetails"
+              cards: "#/components/schemas/NpgAuthRequestDetails"
```

Comment:
I would rename this field into something specific to the method rather than the gateway, in the future we will have other methods always on npg.
Maybe CardsAuthRequestDetails?

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/293/commits/b6b488b026f689a32d155fc91d368e9ee0135b5d